src/components/balances/AssetsTable/index.tsx

```diff
@@ -247,11 +259,55 @@ const AssetsTable = ({
         showHiddenAssets={showHiddenAssets}
       />
 
-      <div className={css.container}>
-        <EnhancedTable rows={rows} headCells={headCells} />
-      </div>
+      {hasNoAssets ? (
+        <NoAssets />
+      ) : (
+        <div className={css.container}>
+          <EnhancedTable rows={rows} headCells={headCells} />
+        </div>
+      )}
     </>
   )
 }
 
+const NoAssets = () => {
+  const router = useRouter()
+  const safeAddress = useSafeAddress()
+  const chain = useCurrentChain()
+  const settings = useAppSelector(selectSettings)
+  const qrPrefix = settings.shortName.qr ? `${chain?.shortName}:` : ''
+  const qrCode = `${qrPrefix}${safeAddress}`
+
+  return (
+    <Paper>
+      <Grid container gap={3} alignItems="center" justifyContent="center" py={10} px={2}>
+        <Grid item>
+          <Box p={2} border="1px solid" borderColor="border.light" borderRadius={1}>
+            <QRCode value={qrCode} size={195} />
+          </Box>
+        </Grid>
+        <Grid item container xs={12} md={6} gap={2} flexDirection="column">
+          <Typography variant="h3" fontWeight="bold">
+            Add funds to get started
+          </Typography>
+          <Typography>
+            Add funds directly from your bank account or copy your address to send tokens from a different account.
+          </Typography>
+          <Box bgcolor="background.main" p={2} borderRadius="6px" alignSelf="flex-start" fontSize="14px">
+            <EthHashInfo address={safeAddress} shortAddress={false} showCopyButton hasExplorer avatarSize={24} />
+          </Box>
+          <Box alignSelf="flex-start">
+            {/* TODO: Insert link for Ramp app */}
```

Comment:
In order to find the app we should probably add a tag "on-ramping" to the Safe App so we can search for the appUrl by tag similar to the Safe Governance App links in the header.